README.md

```diff
@@ -77,6 +77,7 @@ Know of another Hyper package? [Help add it!](https://github.com/bnb/awesome-hyp
 * [hyper-arc-dark-controls](https://www.npmjs.com/package/hyper-arc-dark-controls) - Pretty window controls from the Arc Dark theme for Hyper.
 * [hyper-mac-controls](https://www.npmjs.com/package/hyper-mac-controls) - Mac-like window controls for Hyper.
 * [hyper-terminal-tabs](https://www.npmjs.com/package/hyper-terminal-tabs) - Brings Terminal.app look and feel to Hyper.
+* [hyper-dark-scrollbar](https://github.com/moso/hyper-dark-scrollbar) - Pretty scrollbar for Hyper.
```

Comment:
The great majority of the existing links point to [npmjs.com](https://www.npmjs.com/), but I'm not sure if it has to be enforced.
The `CONTRIBUTING.md` doesn't have an explicit rule for this.

Calling @iamstarkov and @bnb for clarification :)